src/LazyImage/src/BlurHash/FetchImageContent.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LazyImage\BlurHash;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class FetchImageContent
+{
+    public function __invoke(string $filename): string
+    {
+        return file_get_contents($filename);
```

Comment:
You risk the TypeError here, if file_get_contents fails it will return "false", not following the "string" return type.

I'd add a if there and throw an exception if the image is not "gettable"

Reply:
Yup you're right!

What about adding `mixed` and let Intervention does the verification?

Reply:
It's more the job of the "fetch_image_content" i think, no ?

Reply:
In fact that's not really doable, Intervention type with `mixed` and let the correct handler handle the value (SplFileInfo, Base64, Color, ...):  
<img width="235" alt="image" src="https://github.com/symfony/ux/assets/2103975/79242383-c537-447c-872c-84d3b4bc3c5e">